Sources/MapboxNavigation/Resources/pl.lproj/Localizable.strings

```diff
@@ -2,275 +2,274 @@
+"INCORRECT_SPEED_LIMIT" = "Nieprawidłowy limit prędkości";
 
 /* Specific route feedback that an exit was incorrect. */
-"INCORRECT_VISUAL_EXIT_INFO_INCORRECT_FEEDBACK" = "Exit info incorrect";
+"INCORRECT_VISUAL_EXIT_INFO_INCORRECT_FEEDBACK" = "Niepoprawne informacje o wyjeździe";
 
 /* General category of route feedback where visual instruction was incorrect. */
-"INCORRECT_VISUAL_FEEDBACK" = "Looks incorrect";
+"INCORRECT_VISUAL_FEEDBACK" = "Wygląda niepoprawnie";
 
 /* Specific route feedback that an instruction was missing. */
-"INCORRECT_VISUAL_INSTRUCTION_MISSING_FEEDBACK" = "Instruction missing";
+"INCORRECT_VISUAL_INSTRUCTION_MISSING_FEEDBACK" = "Brak instrukcji";
 
 /* Specific route feedback for an unnecessary instruction. */
-"INCORRECT_VISUAL_INSTRUCTION_UNNECESSARY_FEEDBACK" = "Instruction unnecessary";
+"INCORRECT_VISUAL_INSTRUCTION_UNNECESSARY_FEEDBACK" = "Niepotrzebna instrukcja";
 
 /* Specific route feedback that the wrong lane was specified. */
-"INCORRECT_VISUAL_LANE_GUIDANCE_INCORRECT_FEEDBACK" = "Lane guidance incorrect";
+"INCORRECT_VISUAL_LANE_GUIDANCE_INCORRECT_FEEDBACK" = "Niepoprawna informacja o pasie";
 
 /* Specific route feedback that a maneuver specified was incorrect. */
-"INCORRECT_VISUAL_MANEUVER_INCORRECT_FEEDBACK" = "Maneuver incorrect";
+"INCORRECT_VISUAL_MANEUVER_INCORRECT_FEEDBACK" = "Niepoprawny manewr";
 
 /* Specific route feedback for incorrect street name. */
-"INCORRECT_VISUAL_STREET_NAME_INCORRECT_FEEDBACK" = "Street name incorrect";
+"INCORRECT_VISUAL_STREET_NAME_INCORRECT_FEEDBACK" = "Niepoprawna nazwa ulicy";
 
 /* Specific route feedback for incorrect turn arrow being shown. */
-"INCORRECT_VISUAL_TURN_ICON_INCORRECT_FEEDBACK" = "Turn icon incorrect";
+"INCORRECT_VISUAL_TURN_ICON_INCORRECT_FEEDBACK" = "Niepoprawna ikona skrętu";
 
 /* Format for displaying the first two major ways */
-"LEG_MAJOR_WAYS_FORMAT" = "%1$@ and %2$@";
+"LEG_MAJOR_WAYS_FORMAT" = "%1$@ i %2$@";
 
 /* Format string for a short distance or time less than a minimum threshold; 1 = duration remaining */
 "LESS_THAN" = "<%@";
 
 /* Alternatives selection note about a longer route distance in any unit. */
-"LONGER_ALTERNATIVE" = "%@ longer";
+"LONGER_ALTERNATIVE" = "%@ dłuższa";
 
 /* Title for button that cancels user's submission of feedback on navigation session issues. */
-"NAVIGATION_REPORT_CANCEL" = "Cancel";
+"NAVIGATION_REPORT_CANCEL" = "Anuluj";
 
 /* Title for button that submits user's feedback on multiple navigation session issues. 1 is the number of items */
-"NAVIGATION_REPORT_ISSUES" = "Send %ld Item(s)";
+"NAVIGATION_REPORT_ISSUES" = "Wyślij %ld element(ów)";
 
 /* Accessibility value of label indicating the absence of a rating */
-"NO_RATING" = "No rating set.";
+"NO_RATING" = "Brak ustawionej oceny.";
 
 /* Specific route feedback that user positioning is incorrect. */
-"PASSIVE_NAVIGATION_FEEDBACK_BAD_GPS" = "Bad GPS";
+"PASSIVE_NAVIGATION_FEEDBACK_BAD_GPS" = "Zły GPS";
```

Comment:
Pobably will be better:
`Brak GPS`

Reply:
Direct translation of Ilya suggestion is: "Missing GPS"
English translation seems also lil bit incorrect.
I would do 
"Weak GPS signal"
"Słaby sygnał GPS"